onnxruntime/core/session/provider_registration.cc

```diff
@@ -115,7 +123,7 @@ ORT_API_STATUS_IMPL(OrtApis::SessionOptionsAppendExecutionProvider,
   } else {
     ORT_UNUSED_PARAMETER(options);
     status = OrtApis::CreateStatus(ORT_INVALID_ARGUMENT,
-                                   "Unknown provider name. Currently supported values are 'SNPE', 'XNNPACK', and 'AZURE'");
+                                   "Unknown provider name. Currently supported values are 'OPENVINO', 'SNPE', 'XNNPACK', and 'AZURE'");
```

Comment:
do you mind adding QNN and WEBNN since you are touching this line?

Reply:
Added QNN and WEBNN part of the Error status.

Reply:
thanks!